FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -22,25 +24,38 @@ import {
 } from '../../store/actions/feed';
 import {navigationRef} from '../../navigation/RootNavigation';
 import {useAppSelector} from '../../store/store';
-import {
-  FlashList
-} from '@shopify/flash-list';
+import {FlashList} from '@shopify/flash-list';
 import {styles} from './styles';
-import {LMPost} from '../../../LikeMinds-ReactNative-Feed-UI';
+import {
+  LMHeader,
+  LMIcon,
+  LMImage,
+  LMPost,
+  LMVideo,
+} from '../../../LikeMinds-ReactNative-Feed-UI';
```

Comment:
Is the UI library not hosted yet?

Reply:
No, we have just added it as a submodule